test/src/test/java/org/corfudb/integration/StreamIT.java

```diff
@@ -0,0 +1,68 @@
+package org.corfudb.integration;
+
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.StreamView;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.Random;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * A set integration tests that exercise the stream API.
+ */
+
+public class StreamIT {
+    static String layoutServers;
+
+    @BeforeClass
+    static public void getLayoutServers() throws Exception {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        InputStream input = classLoader.getResourceAsStream("CorfuDB.properties");
+        Properties properties = new Properties();
+        properties.load(input);
+
+        layoutServers = (String) properties.get("layoutServers");
+    }
+
+    @Test
+    public void simpleStreamTest() throws Exception {
+
+        CorfuRuntime rt = new CorfuRuntime(layoutServers).connect();
+        rt.setCacheDisabled(true);
+
+        UUID streamId = UUID.nameUUIDFromBytes("stream1".getBytes());
```

Comment:
You should use CorfuRuntime::getStreamID, since we may change the implementation to retrieve the UUID in the future